sumpy/tools.py

```diff
@@ -123,6 +209,159 @@ def get_closest_cached_mi(self, mi):
                 if (np.array(mi) >= np.array(other_mi)).all()),
             key=lambda other_mi: sum(self.mi_dist(mi, other_mi)))
 
+
+class LaplaceDerivativeTaker(MiDerivativeTaker):
```

Comment:
Add a comment on what math this implements (and the complexity implications).

Reply:
Done in 113c9643507884756aa3dfa8782b46fbb87652e7